components/style/counter_style/mod.rs

```diff
@@ -396,7 +396,7 @@ pub enum Symbol {
     /// <string>
     String(String),
     /// <ident>
-    Ident(String),
+    Ident(CustomIdent),
```

Comment:
It'd be nice to add a test-case / extend the test-case from #20171 to account for this, if you feel like it :)